dashboards/jetstream/jetstream-overview.json

```diff
@@ -0,0 +1,1004 @@
+{
+  "displayName": "JetStream Prometheus Overview",
```

Comment:
This is the only place the s in JetStream is capitalized, should this + images be updated, or yaml files?

Reply:
Good catch! Based on:
- https://cloud.google.com/kubernetes-engine/docs/tutorials/serve-gemma-tpu-jetstream
- https://github.com/AI-Hypercomputer/JetStream

It looks like "JetStream" is right, so I've fixed everything else to match.